styles/foobar-ui.min.css

```diff
@@ -1 +1 @@
oup,select,textarea{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=button],[type=reset],[type=submit],button{-webkit-appearance:button}[type=button]::-moz-focus-inner,[type=reset]::-moz-focus-inner,[type=submit]::-moz-focus-inner,button::-moz-focus-inner{border-style:none;padding:0}[type=button]:-moz-focusring,[type=reset]:-moz-focusring,[type=submit]:-moz-focusring,button:-moz-focusring{outline:1px dotted ButtonText}fieldset{padding:.35em .75em .625em}legend{box-sizing:border-box;color:inherit;display:table;max-width:100%;padding:0;white-space:normal}progress{vertical-align:baseline}textarea{overflow:auto}[type=checkbox],[type=radio]{box-sizing:border-box;padding:0}[type=number]::-webkit-inner-spin-button,[type=number]::-webkit-outer-spin-button{height:auto}[type=search]{-webkit-appearance:textfield;outline-offset:-2px}[type=search]::-webkit-search-decoration{-webkit-appearance:none}::-webkit-file-upload-button{-webkit-appearance:button;font:inherit}details{display:block}summary{display:list-item}template{display:none}[hidden]{display:none}.nobullets{list-style:none;margin:0;padding:0}.flex-row{display:flex;flex-flow:row wrap}.alert{font-size:var(--font-size-sm);font-weight:var(--font-weight-regular);padding:var(--space-md) var(--space-xl);display:flex;align-items:center;border-radius:12px;width:100%}.alert-success{background-color:var(--color-success);color:var(--color-success-text)}.alert-error{background-color:var(--color-danger);color:var(--color-danger-text)}.alert-warning{background-color:var(--color-warning);color:var(--color-warning-text)}.alert-info{background-color:var(--color-info);color:var(--color-info-text)}.avatar{display:inline-block;vertical-align:middle;border-radius:var(--border-radius-circle);background-color:var(--color-secondary);color:var(--color-black);font-weight:500;display:flex;justify-content:center;align-items:center}.avatar-sm{height:var(--space-lg);width:var(--space-lg);font-size:var(--space-xs)}.avatar-md{height:var(--space-xl);width:var(--space-xl);font-size:var(--space-sm)}.avatar-lg{height:var(--space-xxl);width:var(--space-xxl);font-size:var(--space-md)}.avatar__img{width:100%;height:100%;object-fit:cover;border-radius:var(--border-radius-circle)}.badge{position:relative;font-size:var(--font-size-md)}.badge-anchor{position:absolute;top:calc(var(--font-size-xs) * -1);right:calc(var(--font-size-sm) * -1);font-size:calc(var( --font-size-md) / 2);border-radius:calc(var(--space-xl)/ 2);line-height:calc(var( --space-xl) / 2);height:calc(var(--space-xl)/ 2);min-width:calc(var(--space-xl)/ 2);background-color:var(--color-primary);color:var(--color-white);padding:0 6px}.badge-anchor-avatar{position:absolute;bottom:0;right:var(--space-xs);border-radius:calc(var(--space-xs)/ 2);line-height:calc(var( --space-xs) / 2);height:var(--space-xs);width:var(--space-xs);background-color:var(--color-secondary);color:var(--color-white);padding:0 4px}
```

Comment:
give proper indentation and alignment to the code blocks.
this is unreadable

Reply:
Hi @Sameony , this is actually minified file.